elasticsearch/src/test/java/org/hibernate/search/elasticsearch/test/DefaultElasticsearchClientFactoryTest.java

```diff
@@ -310,7 +310,7 @@ public void discovery() throws Exception {
 		SearchConfigurationForTest configuration = new SearchConfigurationForTest()
 				.addProperty( CLIENT_PROPERTY_PREFIX + ElasticsearchEnvironment.SERVER_URI, httpUrlFor( wireMockRule1 ) )
 				.addProperty( CLIENT_PROPERTY_PREFIX + ElasticsearchEnvironment.DISCOVERY_ENABLED, "true" )
-				.addProperty( CLIENT_PROPERTY_PREFIX + ElasticsearchEnvironment.DISCOVERY_REFRESH_INTERVAL, "1" );
+				.addProperty( CLIENT_PROPERTY_PREFIX + ElasticsearchEnvironment.DISCOVERY_REFRESH_INTERVAL, "2" );
```

Comment:
Because if the refresh occurs less often, it will be less likely to occur between our two requests (the two calls to `doPost`), which will still execute just as fast.
Anyway, that was just to be safe, but you're right, polling will be more efficient.